### PR TITLE
Extra options

### PR DESCRIPTION
**Pull request by andrii-maglovanyi**
feat: allowDuplicates option
feat: trimSpaces option

### CODE REVIEW

File: src/plugins/tokenizer.js [hunk 1]
```diff
@@ -11,20 +11,28 @@ function defaultTokenizer(input, selection, createToken, options) {
             return token ? { id: token, text: token } : null;
         };
 
+    var duplicates = options.allowDuplicates;
```
Comment (arendjr) [inline src/plugins/tokenizer.js]: Please call the variable `allowDuplicates` too. It's a boolean option, but `duplicates` would indicate it's a list of duplicates.

File: src/plugins/tokenizer.js [hunk 2]
```diff
@@ -11,20 +11,28 @@ function defaultTokenizer(input, selection, createToken, options) {
...
 
     function takeToken(input) {
-        for (var i = 0, length = input.length; i < length; i++) {
-            if (separators.indexOf(input[i]) > -1) {
-                return { term: input.slice(0, i), input: input.slice(i + 1) };
+        var trimmedInput = trim ? input.trim() : input;
+        for (var i = 0, length = trimmedInput.length; i < length; i++) {
+            if (separators.indexOf(trimmedInput[i]) > -1) {
+                var _term = trimmedInput.slice(0, i);
+                var _input = trimmedInput.slice(i + 1);
```
Comment (arendjr) [inline src/plugins/tokenizer.js]: Why the underscores?

Comment (andrii-maglovanyi) [reply src/plugins/tokenizer.js]: just personal preference, renamed variables

**Comment by arendjr** [comment]
Great, thanks!

**Pull request merged**